draft-barnes-acme.md

```diff
@@ -133,6 +135,22 @@ Account Key Pair:
 Recovery Token:
 : A secret value that can be used to demonstrate prior authorization for an identifier, in a situation where all Subject Private Keys and Account Keys are lost.
 
+## Threat model
+
+For most communications between the client and the server, we assume the Internet
+Threat Model {{RFC3552}}, i.e. an attacker that can completely control messages
+between client and server. We additionally assume the attacker can read and
+modify TLS connections between client and server. This allows ACME deployment
+behind a TLS-terminating CDN, for DDoS prevention and other attack mitigation.
+
+For domain validation requests initiated by the server, by necessity we assume a less
+powerful attacker: the "off-path" attacker described in RFC3552 {{RFC3552}}. This
+attacker can send arbitrary messages to the ACME server, but cannot receive
+or respond to arbitrary messages sent by the server. This is the common threat
+model assumed by most domain-validating certificate authorities today.
```

Comment:
I tend to agree with @bifurcation about this. You could say if you want that the system is only secure in an off-path-only threat model